### ISSUE TITLE
Readme example broken: cannot find numbro

### ISSUE THREAD

**Issue by timwis** [handsontable/react-handsontable#17]
Hi folks, I'm trying to use this module but it seems that the `handsontable` module isn't able to resolve `numbro`. [Here's a live demo](https://www.webpackbin.com/bins/-Ki4tGwhr3yny6eIJHpw) of the example used in this library's readme (I added the actual render at the bottom). If you look at the log you'll see an error that it `Cannot find module 'numbro'`.

Am I doing it wrong? (I copied & pasted from the readme)

Ideally, it would be great if (a) there were a way to import a non-compiled/minified version of handsontable (which webpack otherwise throws warnings about), and (b) if handsontable would declare its own dependencies so we could simply `import` one library. But short of that, I'd appreciate your help in getting the example working as intended. Thanks!

**Comment by zekedroid** [comment]
+1, same issue

**Comment by wkopen** [comment]
+1

**Comment by nuty** [comment]
i had same problem. there is ugly way.
see this https://github.com/handsontable/handsontable/issues/3627。

**Comment by nupurgrover** [comment]
I ran into the same issue while trying out Handsontable, and was able to solve it by following instructions [here](https://github.com/handsontable/react-handsontable/wiki/Installation-guide#using-webpack-and-babel).

I agree that it would be much more elegant if `handsontable` declared it's own dependencies.

**Comment by sassanh** [comment]
It's just a nightmare to use this library with browserify, I don't know about webpack. Never had this much trouble with any other library.

**Comment by zekedroid** [comment]
Agreed, it's a little behind its time in terms of deployment strategies. It's a great library otherwise.

**Comment by sassanh** [comment]
After hours of fighting I ended up replacing all `require("handson")` with `require("handsontable/dist/handsontable.full")` in `node_modules/react-handsontable/dist/react-handsontable.js`.
I hope it'll get fixed soon, I hate it to do the same thing in my production machine and forking and fixing and building is just another nightmare.

**Comment by jansiegel** [comment]
Since version `0.3.0` you don't have to import anything except `react-handsontable`, the dependencies should work out of the box: https://www.webpackbin.com/bins/-KpJVuo5Ge6fVA3-KMEk

I'll update the repo's README as soon as possible.

**Comment by sassanh** [comment]
That's good news. Thanks.

**Comment by sassanh** [comment]
Now I got this error:
```
Error: Cannot find module '!!../../css-loader/index.js!./handsontable.full.css' from '/Users/sassanharadji/Projects/MHelli/mhelli/Frontend/node_modules/react-handsontable/dist'
```
Installing `css-loader` leads to this error:
```
Error: Cannot find module '!!../../css-loader/index.js!./handsontable.full.css' from '/Users/sassanharadji/Projects/MHelli/mhelli/Frontend/node_modules/react-handsontable/dist'
```

**Comment by jansiegel** [comment]
Could you provide your webpack config?

**Comment by sassanh** [comment]
I'm not using webpack, I'm using browserify. This is related stuff: https://github.com/sstur/react-rte/issues/32

**Comment by trcosner** [comment]
I am having this same issue. I'm also using browserify. It looks like not all dependencies are being installed. react-handsontable/node_modules only contains pikaday.

**Comment by sassanh** [comment]
I doubt it's related to dependencies, seems like the files in dist folder are in a format that is only recognized by webpack. I guess `'!!../../css-loader/index.js!./handsontable.full.css'` is webpack stuff to load css inside js. Maybe it's alright for a web application to have such things but I guess a library should keep things standard and not rely on a specific compiler. Btw none of the 60+ js libraries I'm using in my project have this kind of problems.

**Comment by trcosner** [comment]
A work around has been to import via
import HotTable from 'react-handsontable/dist/react-handsontable.min.js';
It seems to be working fine this way.

**Comment by michaelwooley** [comment]
@trcosner's  solution worked for me but you must use the `import HotTable from 'react-handsontable/dist/react-handsontable.min.js';` notation for some reason, _not_ `const {HotTable} = require('react-handsontable/dist/react-handsontable.min.js')`,.
